modules/elementor-counter/module.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Elementor\Modules\ElementorCounter;
+
+use Elementor\Core\Isolation\Elementor_Counter_Adapter_Interface;
+use Elementor\Core\Isolation\Wordpress_Adapter;
+use Elementor\Core\Isolation\Wordpress_Adapter_Interface;
+use Elementor\Core\Base\Module as BaseModule;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class Module extends BaseModule implements Elementor_Counter_Adapter_Interface {
+	const EDITOR_COUNTER_KEY = 'e_editor_counter';
+
+	private ?Wordpress_Adapter_Interface $wordpress_adapter = null;
+	private static $should_count_editor = true;
+
+	public function get_name() {
+		return 'elementor-counter';
+	}
+
+	public function __construct( ?Wordpress_Adapter_Interface $wordpress_adapter = null ) {
+		parent::__construct();
+		$this->wordpress_adapter = $wordpress_adapter ?? new Wordpress_Adapter();
+
+		if ( self::$should_count_editor ) {
+			add_action( 'elementor/editor/init', function () {
+				$this->increment( self::EDITOR_COUNTER_KEY );
```

Comment:
Does your other calls to `elementor/editor/init` have an execution order dependancy? Like, does set_user_preferences() have to run before increment()? If so, maybe add priority?

Reply:
Good!!!